tests/storages_tests/test_storages.py

```diff
@@ -768,6 +771,44 @@ def test_get_all_trials(storage_mode: str) -> None:
             storage.get_all_trials(non_existent_study_id)
 
 
+@pytest.mark.parametrize("storage_mode", STORAGE_MODES_GRPC)
+@pytest.mark.parametrize("ttl_cache_seconds", [None, 1.0])
+def test_get_all_trials_grpc_proxy_server_ttl_cache(
+    storage_mode: str, ttl_cache_seconds: float | None
+) -> None:
```

Comment:
I am concerned that there are two if statements inside the test depending on whether ttl_cache_seconds is None or not, so why not separate the test functions for cases where ttl_cache_seconds is None and 1.0?

Reply:
thanks, I will separate them